docs/en/latest/admin-api.md

```diff
@@ -122,24 +122,34 @@ By default, the Admin API checks for references between resources and will refus
 
 You can make a force deletion by adding the request argument `force=true` to the delete request, for example:
 
+:::note
+
+You can get the get the admin_key from config.yaml like this.
+
+```bash
+ admin_key=$(yq '.deployment.admin.admin_key[0].key' conf/config.yaml | sed 's/"//g')
+```
+
+:::
+
 ```bash
-$ curl http://127.0.0.1:9180/apisix/admin/upstreams/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '{
+$ curl http://127.0.0.1:9180/apisix/admin/upstreams/1 -H "X-API-KEY: $admin_key" -X PUT -d '{
```

Comment:
Wouldn't this be interpreted as string literal? Should it be `${admin_key}` instead? Because I saw you save the admin key to an env var as well